sheet_cleaner/main.py

```diff
@@ -12,10 +13,15 @@
 from constants import *
 from functions import *
 
-configfile = '/home/tbrewer/nCoV2019/code/sheet_cleaner/.CONFIG'
+parser = argparse.ArgumentParser(
+    description='Cleanup sheet and output generation script')
+parser.add_argument('--config_file', type=str, default="CONFIG",
```

Comment:
might be convenient to add a short option, as in `parser.add_argument('-c', '--config_file', type=str, #...`